lib/plugins/package/package.js

```diff
@@ -75,6 +79,19 @@ class Package {
       },
     };
   }
+
+  deleteNullEnvVars() {
```

Comment:
This could be put into a separate file in the lib folder, so that the index file only does the initialization of the package plugin and the action implementations are outsourced (like it is done for the other actions - packageService and zipService).